packages/fxa-payments-server/src/routes/Subscriptions/index.tsx

```diff
@@ -158,6 +158,17 @@ export const Subscriptions = ({
     );
   }
 
+  // If the customer has no subscriptions, redirect to the settings page
+  if (
+    customerSubscriptions.length === 0 ||
+    (customer.error &&
+      customer.error.errno === AuthServerErrno.UNKNOWN_SUBSCRIPTION_CUSTOMER)
```

Comment:
This condition will never be evaluated since it's already up in the `if` on line 146.  I think we can remove that `if` block above.

Reply:
The other `if` block checks if there's an error, but _not_ an `UNKNOWN_SUBSCRIPTION_CUSTOMER` error in order to show an error message. 

From what I understand, that specific error code is another way to say the user has no subscriptions-- maybe a bit redundant if `customerSubscriptions` will always be 0 in that case. 

I can remove it if you want.

Reply:
(Sorry for the late response; been busy.)

You're right, I misread that comparison in the `if` above.  Let's keep this as it is then.  Thanks again!